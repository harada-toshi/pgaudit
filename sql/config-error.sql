\! cp sql/config-error/regress.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sleep 1
\! rm /tmp/config.conf
\! psql contrib_regression -q -a < sql/config-error/config-error.sql
\! egrep '(WARNING|pgaudit)' ./tmp_check/tmp_check.log > results/config-error.out

