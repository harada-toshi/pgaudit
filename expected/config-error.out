WARNING:  Configuration file (/tmp/config.conf) parse error at line     4. current token(bad_param), (syntax error, unexpected IDENTIFIER, expecting FIELD_OUTPUT or END_SECTION)
WARNING:  pgaudit.config_file = /tmp/config.conf    => parse error,All setting is reset in the default value.
LOG:  pgaudit extension initialized
